translations/th.lang

```diff
@@ -0,0 +1,63 @@
+error_title=Error
+error_message=เกิดข้อผิดพลาดขึ้น กรุณาลองอีกครั้ง
+admin_message=คุณต้องเป็นแอดมินถึงจะใช้คำสั่งนี้ได้!
+usage_message=วิธีการใช้: 
+success_message=สำเร็จ
+needs_link=ดิสคอร์ดนี้ไม่ได้เชื่อมกับเซิร์ฟ SCP อยู่ ซึ่งจะทำได้โดยการใช้คำสั่งว่า ``!link <server ID>``
+stats_notpublic=ไม่มีข้อมูลการเล่นของคนนี้ ไม่ก็ข้อมูลถูกปิดไว้ ผู้ใช้ต้องไปลงทะเบียนที่ https://scpstats.com ก่อนถึงจะเปิดเผยข้อมูลได้
+stats_title=ข้อมูล
+stats_message=ข้อมูลของ 
+stats_field_kills=ฆ่าไป
+stats_field_deaths=ตายไป
+stats_fields_kd=K/D
+stats_fields_playtime=เวลาที่เล่นไป
```

Comment:
These should be stats_field, not stats_fields. Other than that, everything looks good!